### PR TITLE
Transportation-on-vacation HW

### PR DESCRIPTION
**Pull request by Ostafii4uk**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 2) {
+    return days * 40;
+  }
+
+  if (days > 2 && days < 7) {
+    return (days * 40) - 20;
```
Comment (Marta1993-droid) [inline src/calculateRentalCost.js:36]: use DRY approach. to avoid repeating code create variable x = days*40 and use it further.

**Review by Marta1993-droid — CHANGES_REQUESTED** [review]

**Review by Marta1993-droid — APPROVED** [review]
Nive! You can also put discount amount in variable.

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,19 @@
...
 function calculateRentalCost(days) {
-  // write code here
+  const rentalDaysPrice = days * 40;
+
+  if (days <= 2) {
+    return rentalDaysPrice;
+  }
+
+  if (days > 2 && days < 7) {
+    return rentalDaysPrice - 20;
```
Comment (Ihor-Karpyn) [inline src/calculateRentalCost.js:38]: What mean 20? Look like magic number. Create var

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,19 @@
...
+  }
+
+  if (days >= 7) {
+    return rentalDaysPrice - 50;
```
Comment (Ihor-Karpyn) [inline src/calculateRentalCost.js:42]: same

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 2 above)
Comment (Ostafii4uk) [reply src/calculateRentalCost.js:38]: Ok, got it

File: src/calculateRentalCost.js (hunk 3 above)
Comment (Ostafii4uk) [reply src/calculateRentalCost.js:42]: ok

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**